mat64/list_test.go

```diff
@@ -66,6 +66,11 @@ func isAnySize2(ar, ac, br, bc int) bool {
 	return true
 }
 
+// isAnyVector returns true for any row or column vector sizes.
```

Comment:
I think this should just check if the number of columns is 1. *Vector is always a column.